lightbulb-challenge-sammy/src/components/LightSwitch.js

```diff
@@ -0,0 +1,28 @@
+import React, {useState} from 'react'
+
+// Created LightSwitch logic component
+const LightSwitch = () => {
+
+    // State Values set at false = light switch 'Off'
+    const [lightSwitchStatus, setLightSwitchStatus] = useState(false)
+
+    // Function that updates State Values (sets it value to either true / false)
+    const lightsOnOff = () => {
+        setLightSwitchStatus(lightSwitchStatus === false ? true : false)  
+    }
+    
+    // if light switch is true = 'On', else 'Off'
+    let switchOnOff = lightSwitchStatus === true ? 'On' : 'Off'
+    // if light switch is true = 'yellow', else 'white'
+    let bulbOnOff = lightSwitchStatus === true ? 'yellow' : 'white'
```

Comment:
Nice use of the ternary operators for the switch status and background color! And very clean again!